### ISSUE TITLE
[Request] Android version

### ISSUE THREAD

**Issue by shadowofdarkness** [wwwwwwzx/SMEncounterRNGTool#40]
Would it be possible to port this to Android phones please? I would like to be able to RNG when away from home like at work during my break.

I did see a thing online about the ability to compile mono apps for Android so it may not be to hard.

I was going to try compiling the source for Android but the monodevelop plug-in is for Windows and Mac only and doesn't like Linux so that left me unable to try.

A link to info about compiling for Android
https://www.xamarin.com/

**Comment by wwwwwwzx** [comment]
I will give it a try.

**Comment by wwwwwwzx** [comment]
Btw, I see your name in the reddit discussion of in-depth mechanics on SOS chaining.
Can you tell me how to find the original code using IDA, I'm in trouble with perfect ivs determination for sos chaining

**Comment by shadowofdarkness** [comment]
Thanks for trying to port to Android.

For my name being credited in that Reddit discussion, if it's the one I'm thinking of it was just because I was the first to realize and post that the chain counter was a u8 counter and reset at 255. Which I found through normal gameplay and reasoning of why my chain that I ended at 284 only gave me a 2IV Pokemon.

Sadly I'm not that good at coding and can't help with your IV determination problem.
